.travis.yml

```diff
@@ -29,20 +29,22 @@ addons:
     - xmlto
     - asciidoc
 
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+    - os: osx
+      compiler: clang
+
+before_install:
+  - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew tap homebrew/science; brew install swig tbb muparser R openblas; export HOMEBREW_PREFIX=`brew --prefix`; fi
```

Comment:
* Replace == by =
 * When using multiple instructions, either add "set -e" or replace semicolons by && so that execution stops as soon as an error is detected
 * Indent multilines

IMHO, this is much more readable and easier to maintain:
```
 script:
  - if test "$TRAVIS_OS_NAME" = "linux"; then
        cmake -DCMAKE_INSTALL_PREFIX=~/.local \
              -DSPHINX_EXECUTABLE=~/.local/bin/sphinx-build \
              -DSWIG_EXECUTABLE=~/.local/bin/swig \
              -DHMAT_DIR=~/.local/lib/cmake/hmat \
              -DNLOPT_LIBRARY=~/.local/lib/libnlopt.so \
              -DNLOPT_INCLUDE_DIR=~/.local/include \
              . &&
        make install -j4;
    fi
  - if test "$TRAVIS_OS_NAME" = "osx"; then
        cmake -DCMAKE_INSTALL_PREFIX=~/.local \
              -DSWIG_EXECUTABLE=~/.local/bin/swig \
              -DCMAKE_MACOSX_RPATH=ON \
              -DHMAT_DIR=~/.local/lib/cmake/hmat \
              -DLAPACKE_FOUND=ON \
              -DOPENTURNS_LIBRARIES="-framework Accelerate" \
              -DNLOPT_LIBRARY=~/.local/lib/libnlopt.dylib \
              -DNLOPT_INCLUDE_DIR=~/.local/include \
              -DUSE_SPHINX=OFF \
              -DPYTHON_EXECUTABLE=/usr/bin/python \
              -DPYTHON_INCLUDE_DIR=/usr/include/python2.7 \
              -DPYTHON_LIBRARY=/usr/lib/libpython2.7.dylib \
              -DUSE_COTIRE=ON \
              -DCOTIRE_TESTS=OFF \
              -DCOTIRE_MAXIMUM_NUMBER_OF_UNITY_INCLUDES="-j4" \
              . &&
        make python_unity -j4 &&
        make install/fast;
    fi
```